controllers/storagesystem_controller.go

```diff
@@ -261,12 +275,38 @@ func (r *StorageSystemReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	}
 
+	enqueueStorageSystemRequest := handler.EnqueueRequestsFromMapFunc(
+		func(ctx context.Context, obj client.Object) []reconcile.Request {
+			logger := log.FromContext(ctx)
+			ssList := &odfv1alpha1.StorageSystemList{}
+			err := r.Client.List(ctx, ssList, client.InNamespace(r.OperatorNamespace))
+			if err != nil {
+				logger.Error(err, "Unable to list StorageSystem objects")
+				return []reconcile.Request{}
+			}
+
+			// Return name and namespace of the StorageSystem object
+			request := []reconcile.Request{}
+			for _, ss := range ssList.Items {
+				request = append(request, reconcile.Request{
+					NamespacedName: types.NamespacedName{
+						Namespace: ss.Namespace,
+						Name:      ss.Name,
+					},
+				})
+			}
+
+			return request
+		},
+	)
```

Comment:
Why are we adding code if we are getting rid of storage system?

Reply:
removed it.